contracts/delegate.md

```diff
@@ -60,6 +60,11 @@ event SetRule(
 );
 ```
 
+| Revert Reason               | Scenario                                                       |
+| :-------------------------- | :------------------------------------------------------------- |
+| `INVALID_PRICE_COEF`     | The priceCoef must be greater than 0.                 |
+
+
 ### Price Calculations
 
 All amounts are in the smallest unit \(e.g. wei\), so all calculations based on price result in a whole number. For calculations that would result in a decimal, the amount is automatically floored by dropping the decimal. For example, a price of `5.25` and `senderParam` of `2` results in `signerParam` of `10` rather than `10.5`. Tokens have many decimal places so these differences are very small.
```

Comment:
Edit this line to say what the previous PR changed. e.g. `results in `signerParam` of `10` rather than `10.5`.` is now wrong